.env.example

```diff
@@ -191,3 +191,5 @@ OTEL_METRIC_EXPORT_INTERVAL=5000
 
 # Use OTEL_SERVICE_NAME to specify service.name
 OTEL_SERVICE_NAME=authgear
+
+OAUTH_DEMO_CREDENTIAL_REDIRECT_URI=http://localhost:3100/noproject/sso/oauth2/callback
```

Comment:
Rename this to `OAUTH_SHARED_CREDENTIALS_REDIRECT_URI`

Reply:
It was change to SHARED_AUTHGEAR_ENDPOINT in later commits